rbac/server/api/users.py

```diff
@@ -156,6 +156,17 @@
         request.app.config.DB_CONN, request.url, user_resource, head_block
     )
 
+@USERS_BP.get("api/users/<user_id>/summary")
+@authorized()
+async def get_user_summary(request, user_id):
+    """This endpoint is for returning summary data for a user, just it's user_id,name, email."""
+    head_block = await utils.get_request_block(request)
+    user_resource = await users_query.fetch_user_resource_summary(
+        request.app.config.DB_CONN, user_id, head_block.get("num")
```

Comment:
using the globally passed connection object is discouraged because it can be expired by rethinkdb. If @nadiabahrami is okay with this then we can merge, but I'd recommend creating a new connection object using:

```
conn = await db_utils.create_connection(
        request.app.config.DB_HOST,
        request.app.config.DB_PORT,
        request.app.config.DB_NAME,
    )
```

before running your queries, and pass `conn` through instead of `request.app.config.DB_CONN` and run `conn.close()` before your return statement. The functions in the utils module expect a closed connection and will reconnect before running their queries.

Thanks!

Reply:
and can you squash/rebase the commits? these all address one issue and I think they can be consolidated into one commit.